src/Config/Config.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PharIo\ComposerDistributor\Config;
+
+use PharIo\ComposerDistributor\FileList;
+
+class Config
+{
+	/** @var string */
+	private $package;
+
+	/** @var string */
```

Comment:
Can't we use an `SPLFile`-object here? Thast way we can more easily check whether the file or folder actually exists and raise an Exception